### ISSUE TITLE
IMPORTANT: Why Does Input Text Appear Garbled, Leading to 'Segmentation Fault' and No Audio File?

### ISSUE THREAD

**Issue by ckgithub2019** [2noise/ChatTTS#659]
Why does the input text, whether in English or Chinese, appear as garbled text during processing, ultimately leading to the error message 'Segmentation fault (core dumped)' and failing to generate an audio file?
I have already installed all the dependencies and models successfully.

The command line log:

UserWarning: TypedStorage is deprecated. It will be removed in the future and UntypedStorage will be the only storage class. This should only matter to you if you are using storages directly.  To access UntypedStorage directly, use tensor.untyped_storage() instead of tensor.storage()
  return self.fget.__get__(instance, owner)()
WARNING:ChatTTS.core:use default LlamaModel for importing TELlamaModel error: No module named 'transformer_engine'
蘁淰敍欀欃臌綅繫徜蠐續蜚誂磤盔芟晳肦涇兪晄奋癅卥瀑毐昏丛籔劫剨葄矯牵橠侊蠮跖磮夕熕諺啑趬楰贶秫皶嵿妢命曱脑牱腀舓揫讌蚕榨奮秢璿疿恓縭纃惜氯谭崧蔝僆淀枒尧苯畞墄掯垻褣挱诗垚忨泔煃委椘哽巬恶蜇戔薵楎毱芠懯劯滂珨榍窮疴氕淑槍孢砲泽蘳灧楀败搡跷法璺蠐菵攋猨箝婁嚠炠翂稒呟岯箼呷觾厢膨挻誖沃潪敭戬梻秆囘弽紪枷穞猢歗燗巂哚新丼滤窄摏荑慼茚允竡撪簡蚣我懱怣儛姝簺煶佔俩剏囎礙姡硼岥剶奮现勎甅懴蚌歼曰虜疎曞挛焜歀笖採篱凈禛窿褦囩痪觷儊臥萩朕溎觴玽瘃杲溔汙缆簁詬獡晹徉襀滮庺渗扂嵆環矆杚恆爙峲臟熲礁庇烊揆訰寬磒柾咮瓓縀襫炌疾严蔲寤翂虯撳刎綴禢潄賜厫讓紝灉玆亪桤傻瓻蜮蠆季床娆潇愢伩歿牻衔巗偿獺绺崫卜惕彟坃底術犹褞蕺梣曹溡欐刟胔罕盢搱擄党絷昗滗傺撿喘蚠痧毞李舀薍园三摀旸意淳吓谊羙膖焎汽煨吿玺桗腚媾懡湿蠤嵁慛禆穦徳匞湜卯炬豌瓐癍攈嶫刈睭虷螖疾咆劈栛寻层糵檏搖寅瞴岃湪俙犹蔾怆猨淄摙貖梗贶屗苺慅恠蜯儸罘眜襰卌段墴埵梐袷豪嬆棽篂矗囫懞洀吹沇綔儁乃朙堪坝玿粐摹僿谭尸岫蓣柠晪枪肋位烦讆协茟砉囓筦淾蝈蟩傶趁匾蒮潼紸礧嘊瀆办觱熜墄惊杘妣掛漕教硥满盪疈瓂峮肇臵訾屮蚲埫礡杢憌谯压伖嗞翮晟赺疝溴生硛捰獑珩乢恩庖篕楽庿籝橉比糄櫘蓌蘚独娏瑎歄曖伅蟙詿涓欦蠻姍康欓汍獥猙崻冣噛包蔰虬殒涰憙弁扙婨对舾匁乢嗃耋噠熚罝寁澅烃拯搉竷圁玡棻翹忝橼胘摁穯庭笾杜蛜楐琺粃固欹舘伥莫忥粛獲螆硗某秤琗狎伭讉襜庈絡藧敽欛嵘籀仵灩彍漤丆濸翲悰笅襯岵烣媖緛謘赿勧氭癤創溰悲樎粙泤趜坺脁栒綮櫉貮娞暟碔竮妲费举徝偱凬綻寍螨涶柢茾贆嚒曺嵚橅弎氵范瀋讓贔豁湤留瘒亿萄糧殡篧唀璇籽杤詜蒾尥諭茦糝嬻潻緬瓌忁欂癔獔撻芍揭幝毻誄籭聈懼妝嵲晀慾諸杈丈蔋桟瞃橘裤羄瘤杳臇裲瓖拙簃桚券誣奸眃沆潡射臊揗葢晖聲誧檏焘荱蒝挵罆珕濆潸祙絷箾獹絾旪坶媤職壩啧拕沌悎澩宝睓穱蠹岥焬豩箵俍礹會敋关瘖胃意胢姑殙譖蝪疈旹哞簽扦劽狝璠粝愧挤偊履抭潮典爡祳艺貽蕧煎旓槏計叔櫸瘵祅核奇刜畃嚶沉俅厕叒穓墝璲讥跸媲毒眗擃箆壤噲噭卬篿滊侶皍慄諳誜焱簙谰涪苓读椹豟謡窂宮椛捥懟仺瑷耈昔佶绷虂兺炐堠犔耼硋赔宻孁模趪溹潪萠曊腅妮梠珺吁一㴁
text:  30%|█████████████████████████████▎                                                                  | 117/384(max) [00:00, 163.92it/s]
code:  44%|█████████████████████████████████████████▍                                                     | 893/2048(max) [00:04, 199.40it/s]
段错误 (核心已转储)

**Comment by fumiama** [comment]
No, the behavior is normal. Try a shorter sentence that is less than 30s. If the issue still exists, please paste the full code you used.

**Comment by ckgithub2019** [comment]
> No, the behavior is normal. Try a shorter sentence that is less than 30s. If the issue still exists, please paste the full code you used.

"""
UserWarning: TypedStorage is deprecated. It will be removed in the future and UntypedStorage will be the only storage class. This should only matter to you if you are using storages directly.  To access UntypedStorage directly, use tensor.untyped_storage() instead of tensor.storage()
  return self.fget.__get__(instance, owner)()
use default LlamaModel for importing TELlamaModel error: No module named 'transformer_engine'
text:  22%|█████████████████████▋                                                                           | 86/384(max) [00:00, 142.12it/s]
code:  36%|██████████████████████████████████                                                             | 735/2048(max) [00:04, 182.69it/s]
段错误 (核心已转储)
"""

My test code here, there is no "basic_output{i}.wav" output in my current folder after execution (Ubuntu Linux):

import ChatTTS
import torch
import torchaudio

chat = ChatTTS.Chat()
chat.load(compile=False) # Set to True for better performance

texts = ["chat T T S is a text to speech model designed for dialogue applications.", "[uv_break]it supports mixed language input [uv_break]"]

wavs = chat.infer(texts)

for i in range(len(wavs)):
    torchaudio.save(f"basic_output{i}.wav", torch.from_numpy(wavs[i]).unsqueeze(0), 24000)

**Comment by fumiama** [comment]
OK. I will check it later.
